build/serve.js

```diff
@@ -102,7 +102,7 @@ gulp.task('serve:watch', ['watch'], serveDevelopmentMode);
 /**
  * Serves the application in development mode.
  */
-gulp.task('serve', ['index'], serveDevelopmentMode);
+gulp.task('serve', ['spawn-backend','index'], serveDevelopmentMode);
```

Comment:
This wasn't added here because backend would crash with no `apiserver-host` param. Thanks!